engine/src/main/java/io/camunda/zeebe/engine/processing/incident/IncidentRecordWrapper.java

```diff
@@ -105,7 +105,7 @@ public long getRequestId() {
   }
 
   @Override
-  public long getLength() {
+  public int getLength() {
```

Comment:
I switched these things from long to int mostly because we add a bunch of unnecessary casts everywhere. I'm not sure why we ever went with longs - did we expect ByteBuffer to eventually support more than 2GB? Do we want to write buffers > 2GB?